docs/api/ref/DesktopAgent.md

```diff
@@ -334,18 +342,17 @@ await fdc3.raiseIntent(startChat.intent.name, context, selectedApp);
 ```
 
 #### See also
+
 * [`findIntent()`](#findintent)
 * [`ResolveError`](Errors#resolveerror)
 
-
-
 ### `getCurrentChannel`
 
 ```ts
 getCurrentChannel() : Promise<Channel | null>;
 ```
 
-Optional function that returns the `Channel` object for the current User channel membership.  In most cases, an application's membership of channels SHOULD be managed via UX provided to the application by the desktop agent, rather than calling this function directly. 
+Optional function that returns the `Channel` object for the current User channel membership.  In most cases, an application's membership of channels SHOULD be managed via UX provided to the application by the desktop agent, rather than calling this function directly.
```

Comment:
```suggestion
Optional function that returns the `Channel` object for the current User channel membership.  In most cases, an application's membership of channels SHOULD be managed via UX provided to the application by the Desktop Agent, rather than calling this function directly.
```